release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -71,7 +71,7 @@ See xref:../installing/installing_azure_stack_hub/installing-azure-stack-hub-use
 [id="ocp-4-9-upgrade-pause-mhc"]
 ==== Pausing machine health checks before updating the cluster
 
-During the upgrade process, nodes in the cluster might become temporarily unavailable. In the case of worker nodes, the machine health check might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, {product-title} {product-version} introduces `cluster.x-k8s.io/paused=""` annotation to let you pause the `MachineHealthCheck` resources before updating the cluster. 
+During the upgrade process, nodes in the cluster might become temporarily unavailable. In the case of worker nodes, the machine health check might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, {product-title} {product-version} introduces `cluster.x-k8s.io/paused=""` annotation to let you pause the `MachineHealthCheck` resources before updating the cluster.
 
 For more information, see xref:../updating/updating-cluster-cli.adoc#machine-health-checks-pausing[Pausing a MachineHealthCheck resource].
```

Comment:
I know this wasn't written for this PR, but can this xref go directly to 'Pausing a MachineHealthCheck resource`? It currently goes to the top of the page.

Reply:
@abhatt-rh - PTAL. I think this might need the filename after the anchor:
`xref:../updating/updating-cluster-cli.adoc#machine-health-checks-pausing_updating-cluster-cli[Pausing a MachineHealthCheck resource]`

Reply:
Thank you @stevsmit and @bobfuru. I have fixed it the PR here: https://github.com/openshift/openshift-docs/pull/37079. PTAL

Reply:
Thank you @abhatt-rh for the fix and thank you @stevsmit and @bobfuru for the review!